app/views/candidate_interface/volunteering/review/show.html.erb

```diff
@@ -2,22 +2,24 @@
 <% content_for :before_content, govuk_back_link_to(candidate_interface_application_form_path, 'Back to application') %>
 
 <div class="govuk-grid-row">
-  <div class="govuk-grid-column-full">
+  <div class="govuk-grid-column-two-thirds">
     <h1 class="govuk-heading-xl">
       <%= t('page_titles.volunteering.short') %>
     </h1>
 
-    <%= render(CandidateInterface::VolunteeringReviewComponent.new(application_form: @application_form)) %>
-
-    <%= govuk_button_link_to t('application_form.volunteering.another.button'), candidate_interface_new_volunteering_role_path, class: 'govuk-button--secondary' %>
-
-    <%= form_with model: @application_form, url: candidate_interface_complete_volunteering_path do |f| %>
-      <div class='govuk-form-group'>
-        <%= f.hidden_field :volunteering_completed, value: false %>
-        <%= f.govuk_check_box :volunteering_completed, true, multiple: false, label: { text: t('application_form.volunteering.review.completed_checkbox') } %>
-      </div>
-
-      <%= f.govuk_submit t('continue') %>
+    <%= govuk_button_link_to nil, candidate_interface_course_choices_choose_path, class: 'govuk-button--secondary' do %>
```

Comment:
It'd be nice if `govuk_button_link_to` behaved the same as Rails’s [`link_to`](https://api.rubyonrails.org/classes/ActionView/Helpers/UrlHelper.html#method-i-link_to) helper, where you can avoid passing in `nil` if you're specifying the link text as a block. Maybe a dev could help us with this?

Reply:
Yeah, I’ve come across this same issue with the `govuk_link_to` helper, to which I added a `TODO` comment. Will add a `TODO` comment for this helper too.